.github/workflows/build.yml

```diff
@@ -0,0 +1,165 @@
+  GITHUB_EVENT: ${{ github.event_name }}
+  BRANCH_NAME: ${{ github.ref_name }}
+
+jobs:
+  Run:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        stage: [ build , source_clear, e2e_tests ]
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: adopt
+          java-version: 17
+          server-username: MAVEN_USERNAME
+          server-password: MAVEN_PASSWORD
+
+      - name: Setup maven
+        shell: bash
+        run: |
+          [ -d ${HOME}/.m2/ ] || mkdir -p ${HOME}/.m2/
+          cp -v _ci/settings.xml ${HOME}/.m2/ || cp -v .travis.settings.xml ${HOME}/.m2/settings.xml
+
+      - name: Cache the Maven packages to speed up build
+        uses: actions/cache@v1
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+
+      - name: Run ${{ matrix.stage }}
+        if: github.ref_name != 'company_release'
+        run: |
+           bash _ci/init.sh
+           bash _ci/${{ matrix.stage }}.sh
+
+  Release:
+    if: ${{ ( startsWith(github.ref_name, 'HF/')  || startsWith(github.ref_name, 'SP/') || github.ref_name == 'master' )  &&
+        contains(github.event.head_commit.message, '[trigger release]') || inputs.commitMessage == '[trigger release]'  &&
+        github.event_name != 'pull_request' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: adopt
+          java-version: 17
+          server-username: MAVEN_USERNAME
+          server-password: MAVEN_PASSWORD
+
+      - name: Cache the Maven packages to speed up build
+        uses: actions/cache@v1
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+
+      - name: "Login to Quay.io"
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+
+      - name: "Configure AWS credentials"
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_S3_RELEASE_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_S3_RELEASE_SECRET_KEY }}
+          aws-region: eu-west-1
+
+      - name: Run Script Release to nexus
+        run: |
+          bash _ci/init.sh
+          bash _ci/release.sh
+          source _ci/prepare_staging_deploy.sh
+
+      - name: "Release to S3 Staging Bucket"
+        run: | 
+           aws s3 cp --recursive ./deploy_dir_community s3://alfresco-artefacts-staging/community/GoogleDocs/${VERSION}
```

Comment:
How is the value of version going to come?

Reply:
it is exporting the value of version from prepare_staging_deploy.sh script file